iso.bbx

```diff
@@ -58,34 +64,47 @@
   isbn=true,
   doi=true,
   eprint=true,
-  % sorting=nyt,
-  maxnames=9,
-  minnames=1,
-  citetracker=true,
-  autolang=other,
-  date=year,
-  urldate=iso,
-  seconds=true,
+  maxnames=9,% if possible, all creators should be given in the reference
+  minnames=1,% if any names are omitted => first creator + and others (et al.)
+  citetracker=true,% citation tracker in global mode
+  autolang=other,% Use the 'langid' language for the entry
+  date=year,% Use only years
+  urldate=iso,% Use ISO8601 Extended Format (yyyy-mm-dd) for URL 'seen' dates
+  seconds=true,% ISO8601 format requires 'seconds=true'
+  datecirca=true,% Use circa for approximate dates
 }
 
+% Default definitions of beginning and closing macro
+% in bibliography drivers
 \newbibmacro*{begentry}{}
 \newbibmacro*{finentry}{\finentry}
 
-%commands
+% COMMANDS
 
+% Localisation strings in the main document language
 \newcommand\mainsstring[1]{\bibsstring[\mainlang]{#1}}
 \newcommand\mainlstring[1]{\biblstring[\mainlang]{#1}}
 
+% The separator between 'titles' and 'subtitles'
 \renewcommand*\subtitlepunct{\addspacecolon\addspace}
-\renewcommand\multinamedelim{\addsemicolon\addspace}
-\renewcommand\finalnamedelim{\multinamedelim}
 
+% A colon preceded with or without space
 \newcommand\addspacecolon{%
   \iftoggle{bbx:spcolon}
     {\addnbspace\blx@addsppunct{colon}}
     {\unspace\blx@addsppunct{colon}}%
 }
 
+% Context-sensitive delimiters
+
+% The delimiter between mutliple names
+% A semicolon: legacy of older versions of standard / Czech interpretations
+\DeclareDelimFormat{multinamedelim}{\addsemicolon\addspace}
```

Comment:
If preceded by another `\add...` `\addspace` can turn into `\space`.
```suggestion
\DeclareDelimFormat{multinamedelim}{\addsemicolon\space}
```
As mentioned above this has the potential for backwards compatibility issues. But I for one think progress is a good thing here.